src/scripts/genEventPipe.py

```diff
@@ -274,7 +274,9 @@ def generateEventPipeCmakeFile(etwmanifest, eventpipe_directory, extern):
 """)
 
 def generateEventPipeHelperFile(etwmanifest, eventpipe_directory, extern):
-    with open_for_update(os.path.join(eventpipe_directory, "eventpipehelpers.cpp")) as helper:
+    eventpipehelpersPath = os.path.join(eventpipe_directory, "eventpipehelpers.cpp")
+    print(eventpipehelpersPath)
```

Comment:
Did you mean to leave these prints here?

Reply:
Yes I did. The way that the python script communicates to CMake which files it generates are by writing the path of the file to stdout, which CMake reads in to get the file paths.